docs/http/migration-guide.md

```diff
@@ -0,0 +1,212 @@
+| `net.protocol.name` &rarr; `network.protocol.name` | Recommended &rarr; Conditionally required if not `http` and `network.protocol.version` is set |
+| `net.protocol.version` &rarr; `network.protocol.version` | Examples fixed: `2.0` &rarr; `2` and `3.0` &rarr; `3`; see note if [migrating from `<= v1.19.0`](#migrating-from--v1190) |
+| New: `error.type` |  |
+<!-- prettier-ignore-end -->
+
+References:
+
+- [Metric `http.client.duration` v1.20.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/metrics/semantic_conventions/http-metrics.md#metric-httpclientduration)
+- [Metric `http.client.request.duration` v1.23.0 (stable)](http-metrics.md#metric-httpclientrequestduration)
+
+### HTTP server duration metric
+
+Metric changes:
+
+- **Name**: `http.server.duration` &rarr; `http.server.request.duration`
+- **Unit**: `ms` &rarr; `s`
+- **Description**: `Measures the duration of inbound HTTP requests.` &rarr;
+  `Duration of HTTP server requests.`
+- **Histogram buckets**: boundaries updated to reflect change from milliseconds
+  to seconds, and zero bucket boundary removed
+- **Attributes**: see table below
+
+<!-- prettier-ignore-start -->
+| Attribute change | Comments |
+| --- | --- |
+| `http.route` | No change |
+| `http.method` → `http.request.method` | Now captures only 9 common HTTP methods by default plus `_OTHER` |
+| `http.status_code` → `http.response.status_code` |  |
+| `http.scheme` → `url.scheme` | Now factors in [`X-Forwarded-Proto` span][X-Forwarded-Proto], [`Forwarded#proto` span][Forwarded#proto] headers |
+| `net.protocol.name` → `network.protocol.name` | Recommended → Conditionally required if not `http` and `network.protocol.version` is set |
+| `net.protocol.version` → `network.protocol.version` | Examples fixed: `2.0` → `2` and `3.0` → `3`; see note if [migrating from `<= v1.19.0`](#migrating-from--v1190) |
+| `net.host.name` → `server.address` | &bullet; Recommended → Opt-In (due to high-cardinality vulnerability since based on HTTP headers)<br>&bullet; Now based only on [`Host` span][Host header], [`:authority` span][HTTP/2 authority], [`X-Forwarded-Host` span][X-Forwarded-Host], [`Forwarded#host` span][Forwarded#host] headers |
+| `net.host.port` → `server.port` | &bullet; Recommended → Opt-In (due to high-cardinality vulnerability since based on HTTP headers)<br>&bullet; Now based only on [`Host` span][Host header], [`:authority` span][HTTP/2 authority], [`X-Forwarded-Host` span][X-Forwarded-Host], [`Forwarded#host` span][Forwarded#host] headers |
+| New: `error.type` |  |
+<!-- prettier-ignore-end -->
+
+References:
+
+- [Metric `http.server.duration` v1.20.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/metrics/semantic_conventions/http-metrics.md#metric-httpserverduration)
+- [Metric `http.server.request.duration` v1.23.0 (stable)](http-metrics.md#metric-httpserverrequestduration)
+
+## Migrating from a version earlier than v1.20.0?
+
```

Comment:
I think it would be helpful to add a quick note that below points are in additional to things listed above